packages/engine/src/common/constants/Axis3D.ts

```diff
@@ -0,0 +1,16 @@
+// Common readonly 3D Axis definitions
+import { Vector3 } from 'three'
+
+/** X Axis (1,0,0) */
+export const X = Object.freeze(new Vector3(1, 0, 0))
```

Comment:
let's rename these to `xAxis`, `rightDirection` etc to be a little more descriptive